src/index.tsx

```diff
@@ -3,7 +3,24 @@ import Primitives from "src/components/Primitives";
 import Scroll from "src/components/Scroll";
 import Slider from "src/components/Slider";
 import Viewer from "src/components/Viewer";
+import {
+  parseAnnotationTarget,
+  type AnnotationTargetExtended,
+} from "src/lib/annotation-helpers";
+import { createOpenSeadragonRect } from "src/lib/openseadragon-helpers";
+import { type Plugin, type PluginInformationPanel } from "src/types/plugins";
 
-export { Image, Primitives, Scroll, Slider, Viewer };
+export {
+  Image,
+  Primitives,
+  Scroll,
+  Slider,
+  Viewer,
+  parseAnnotationTarget,
+  type AnnotationTargetExtended,
+  createOpenSeadragonRect,
+  type Plugin,
+  type PluginInformationPanel,
```

Comment:
I also wonder if there's cleaner way to handle these exports, or group or title them to clarify their context to future consumers of the package?